Directory.Build.props

```diff
@@ -361,6 +349,16 @@
     <RefPath Condition="$(TargetGroup.EndsWith('aot'))">$(RefRootPath)$(TargetGroup.TrimEnd('t').TrimEnd('o').TrimEnd('a'))/</RefPath>
     <NetStandardRefPath>$(RefRootPath)netstandard/</NetStandardRefPath>
     <NetFxRefPath>$(RefRootPath)netfx/</NetFxRefPath>
+    <GlobalToolsDir>$(ArtifactsDir)tools/</GlobalToolsDir>
+
+    <!-- Helix proprties -->
+    <OSPlatformConfig>$(OSGroup).$(Platform).$(ConfigurationGroup)</OSPlatformConfig>
+    <AnyOSPlatformConfig>AnyOS.AnyCPU.$(ConfigurationGroup)</AnyOSPlatformConfig>
+    <UnixPlatformConfig>Unix.$(Platform).$(ConfigurationGroup)</UnixPlatformConfig>
```

Comment:
Are AnyOSPlatformConfig and UnixPlatformConfig still being used? Or can we remove them?

Reply:
I see they're still used. I will cleanup once moving to the new Helix model 😄